src/examples.jl

```diff
@@ -323,7 +376,11 @@ PlotExample("Animation with subplots",
 ),
 
 PlotExample("Spy",
-    "For a matrix `mat` with unique nonzeros `spy(mat)` returns a colorless plot. If `mat` has various different nonzero values, a colorbar is added. The colorbar can be disabled with `legend = nothing`.",
+"""
+For a matrix `mat` with unique nonzeros `spy(mat)` returns a colorless plot. If `mat` has 
+various different nonzero values, a colorbar is added. The colorbar can be disabled with 
+`legend = nothing`.
+"""
```

Comment:
You missed a comma here. This makes the Travis tests fail.

Reply:
Thanks for spotting that. I didn't do a very good job of dealing with merge conflicts.